theme/src/layout/index.js

```diff
@@ -12,12 +12,17 @@ import VariantSelect from '../components/variant-select'
 import NavHierarchy from '../util/nav-hierarchy'
 import Details from '../components/details'
 import * as Slugger from '../hooks/use-slugger'
+import {makePreElementsFocusable} from '../util/common'
 
 function Layout({children, pageContext, location}) {
   const {title, description, status, source} = pageContext.frontmatter
 
   const variantRoot = NavHierarchy.getVariantRoot(location.pathname)
 
+  useEffect(() => {
+    makePreElementsFocusable()
+  }, [children])
+
```

Comment:
The `<pre>` elements are rendered from `theme/src/components/code.js`, so I think we can add a `tabIndex={0}` property to those elements instead of doing it asynchronously in a `useEffect`.